packages/common/driver-definitions/src/storage.ts

```diff
@@ -450,3 +461,43 @@ export enum FetchSource {
 	default = "default",
 	noCache = "noCache",
 }
+
+/**
+ * @alpha
+ */
+export interface ISnapshot {
+	snapshotTree: ISnapshotTree;
+	blobContents: Map<string, ArrayBuffer>;
+	ops: ISequencedDocumentMessage[];
+
+	/**
+	 * Sequence number of the snapshot
+	 */
+	sequenceNumber: number | undefined;
+
+	/**
+	 * Sequence number for the latest op/snapshot for the file in ODSP
+	 */
+	latestSequenceNumber: number | undefined;
+
+	snapshotInNewFormat: true;
+}
+
+/**
+ * Additional key in the loader request header
+ * @alpha
+ */
+export enum GetSnapshotOptions {
+	// Indicate scenario in which the snapshot is fetched
+	scenarioName = "scenarioName",
+	// Tell driver to cache the fetched snapshot.
+	cacheSnapshot = "cacheSnapshot",
+}
+
+/**
+ * @alpha
+ */
+export interface ISnapshotFetchOptions {
```

Comment:
I'd add some comments for the structure, and for each field.
It's hard (without looking at the rest of PR) to reason on things like
- what are supported scnearioName values: Is it free-form text that goes to telemetry only? Does it go to telemetry? Does it change behavior?
- cacheSnapshot - I assume this is instruction to driver (that kind of clarification can be here or on getSnapshot() call).

Reply:
Added